src/main.rs

```diff
@@ -221,27 +232,33 @@ mod tests {
     }
 
     impl Guest {
-        fn new() -> Self {
+        fn new_from_ip_class(class: &str, id: u8) -> Self {
```

Comment:
nit: When I see talk about IP classes I think about network classes (e.g. , https://en.wikipedia.org/wiki/Classful_network) but actually this is different ranges. As such I think it would be good to substitute all use of "class" with range.

Reply:
Fixed, thanks.